mantidproject/mantid
# Muon GUI automatically recalculate

**Description of work.**

This updates the new Muon GUI so that the group and pair data is automatically recalculated if the values on the home tab or the group ranges below the grouping table are changed.

**Report to:** Anthony 

**To test:**

Open the new GUI and load some data. Keep an eye on the logs. Try changeing some values on the home tab and check that a recalculation is triggered. 



Fixes #24675 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
